### ISSUE TITLE
We could do a “view my license” page for easy CC license reference

### ISSUE THREAD

**Issue by perpetuatingcuriosity** [creativecommons/list#296]
Users could see an in-app visual representation of the license their photos hold. 

This could be illustrated so it looks like a real document, for example in the style of duolingo certificates/achievement messages: http://pixilateddesign.com/jodycb/wp-content/uploads/2014/01/duolingocertificat.jpg
http://www.theprospect.net/wp-content/uploads/2014/09/Duolingo_Finish.png

So maybe after a user’s first upload, a “View my license” item appears in the drawer menu and it opens up a copy of the license with the user’s name already filled in. Then there’s this easily accessible reference point if users are ever confused about what people can do with their photos.

Maybe we also link to this screen the first time a user uploads a photo to make them aware that it is always available. 

I’m thinking one priority of ours is to help people understand what CC licensing is (we could link to further info about licensing from their page). This could make licensing more visible within the app, and it could also be a point of pride after a first upload: “this is **my** license!” 

What do you think? @sarahpearson : maybe this can help clear up any user’s legal/photo usage questions? #291

**Comment by janeatcc** [comment]
In theory this sounds good. I'd have to see a mock-up though! I like the idea of "View my license" -- but I wonder if to avoid redundancy we could somehow modify/personalize the existing license deed to include User attribution info - and make it all mobile friendly.

**Comment by perpetuatingcuriosity** [comment]
@janeatcc Is there a example of the license in use that you particularly like that I can take a look at?

I was thinking something like this: 
![screen shot 2015-05-20 at 4 51 19 pm](https://cloud.githubusercontent.com/assets/6563209/7736335/131d63d4-ff11-11e4-895d-cc7619f2b932.png)

You would just have easy access a short straightforward explanation with some fun illustration elements and maybe even include a profile pic of the person? 

Then a similar screen would appear when you view the license on someone else’s photo.

**Comment by janeatcc** [comment]
Not on mobile, no. I was thinking more a version of the human-readable deed on mobile (the one that sits at http://creativecommons.org/licenses/by/4.0/), but I just took a look at it on a mobile web browser and it is very far from a simple screen like this.  

I like this, except for the ribbon. I would want to include the CC logo so folks can recognize CC from that logo (it was acquired by MoMA after all, so it carries resonance!). As a future best practice matter, what would it look like for the other licenses that require display of more than one icon? And the text for those would be more lengthy as well. 

What would "Matt Lee" link to (I assume that is why it is underlined).

I would just reiterate that this mobile version of the human-readable deed should carry similar color scheme and/or related elements of the web version... so that it can be somewhat visually consistent across our various expressions of the license. 

If this were to be translated to one licensed object for another mobile platform, could the text be easily changed to "This photo by Matt Lee is licensed under a... " and "Anyone who uses this photo must attribute.."

**Comment by perpetuatingcuriosity** [comment]
This just popped into my head: what if we had an activity log, a log of every interaction the user has with licensed content. So it could read something like: 

Yesterday - You contributed a photo of squirrel, licensed to Matt Lee. 
Yesterday - You contributed a photo of a cat, licensed to Matt Lee. 
Yesterday - You downloaded a photo of a time machine, licensed to TheCCKid. 
Last Week - You bookmarked a photo of an office building, licensed to Ryan Merkley. 


Then say today Matt Lee wants his license to read: mattl. He changes his username and takes a new picture of a train, then the next entry at the top of the log would read:

Today - You contributed a photo of a train, licensed to mattl.

Might be a cool way to make licenses visible and give the user a history of their interactions with content on The List.

Probably too much info for the app in its current state, but might be worth looking into in the future :)

**Comment by janeatcc** [comment]
Hmm, that could be cool, with "licensed" linked to the license?

**Comment by perpetuatingcuriosity** [comment]
Sure, or maybe the person’s name links to their profile, where the license is prominently displayed. 

So in “licensed by mattl”, “mattl” links to matt’s profile where his license is easily accessible.

**Comment by janeatcc** [comment]
Yeah, that could work. Just to clarify though that this would be specifically for The List where one license governs all images. For best practices generally, the licenses may be different so would make more sense to link "licensed" bit.

**Comment by janeatcc** [comment]
Hi @mattl @perpetuatingcuriosity what is the latest on this? I tried the app and this screen isn't in there yet. Is there a latest mock-up I can use? I would like to use it as an example for other mobile apps regarding best practice CC license display/attribution.

**Comment by perpetuatingcuriosity** [comment]
Hey @janeatcc! At the moment we have this issue set for release in 6.0. We’re working on getting MVP ready, so the focus is on #308.

Our current demo is here: http://invis.io/MH3FLVEB7 but doesn’t include specific license examples yet; it’s just the most basic vision for the app. 

I’m a little hesitant to call our above thread a best practice until we’ve really fleshed it out visually and tested it with users. Maybe it’s more of a concept at this stage?

What do you think you would need in a mock up exactly?

**Comment by janeatcc** [comment]
@perpetuatingcuriosity Understood. Yes, sorry, a concept. I think what is above is fine. I just wanted to send the Mozilla folks an example for CC license display in their new BY-SA defaulted Webmaker app. I think our TOS is pretty good for that right now as well. Thanks and let's check back on this once you have a MVP.
